docs/source/installation/colab.rst

```diff
@@ -0,0 +1,19 @@
+Google Colaboratory
+===================
+
+You can install Colab by running the following in a cell:
+
+.. code-block::
+
+   !sudo apt install libcairo2-dev ffmpeg texlive texlive-latex-extra texlive-fonts-extra texlive-latex-recommended texlive-science tipa
+
+Then, run this in a separate cell:
+
+.. code-block::
+   !sudo apt install libpango1.0-dev
+   !pip install manim
+   !pip install IPython --upgrade
```

Comment:
I've just tried in a clean notebook, but can't reproduce the issue with `libpango1.0-dev`. What happened to me, however, was that some packages could not be installed because their mirror information wasn't up-to-date.

Could you please try whether the following suggestion works for you?

```suggestion
.. code-block::

   !sudo apt update
   !sudo apt install libcairo2-dev ffmpeg texlive texlive-latex-extra texlive-fonts-extra texlive-latex-recommended texlive-science tipa libpango1.0-dev

Then, run this in a separate cell:

.. code-block::

   !pip install manim
   !pip install IPython --upgrade
```

Reply:
I've tried running this and it works (in one cell), so I've changed it accordingly.